lib/whenever/version.rb

```diff
@@ -1,3 +1,3 @@
 module Whenever
-  VERSION = '0.9.4'
+  VERSION = '0.9.5'
```

Comment:
Don't bump version in a PR to add something. The bumping of the version is a part of the release process.